packages/jsonld/README.md

```diff
@@ -0,0 +1,104 @@
+# @tangleid/jsonld
+
+Utilities for processing JSON-LD format.
+
+## Installation
+
+Install using [npm](https://www.npmjs.org/):
+
+```shell
+npm install @tangleid/jsonld
+```
+
+or using [yarn](https://yarnpkg.com/):
+
+```shell
+yarn add @tangleid/jsonld
+```
+
+## API Reference
+
+
+* [jsonld](#module_jsonld)
+
```

Comment:
Be aware of the indention.

Reply:
This section is automatically generated by `jsdoc2md`.